app/jobs/metais/sync_related_documents_job.rb

```diff
@@ -0,0 +1,23 @@
+require 'faraday'
+
+class Metais::SyncRelatedDocumentsJob < ApplicationJob
+  queue_as :metais
+
+  API_ENDPOINT = 'https://metais.vicepremier.gov.sk/cmdb/read/relations/neighbours/'
+  
+  def perform(parent)
+    conn = Faraday.new(url: API_ENDPOINT)
+    page_number = 1
+
+    begin
+      response = conn.post(parent.uuid, '{"neighboursFilter":{"usageType":["system","application"],"metaAttributes":{"state":["DRAFT"]},"relType":["CI_HAS_DOCUMENT","Dokument_sa_tyka_KRIS","CONTROL_HAS_DOCUMENT","PROJECT_HAS_DOCUMENT"],"ciType":["Dokument"]},"page":%{page},"perpage":100}' % {page: page_number}, 'Content-Type' => 'application/json')
```

Comment:
Ak je ten request konstanta tak to vytiahni hore ako konstantu nejaky `RELATED_DOCUMENTS_REQUEST_TEMPLATE` alebo nieco take.

Reply:
Cool. Nevedel som, že to vie s tým percentom zmeniť aj v premennej.